recaptcha_enterprise/snippets/util.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import hashlib
+import hmac
+import secrets
+
+
+def _get_hashed_account_id(account_id: str, key: str) -> str:
```

Comment:
Now this needs to be a public module, so `_` prepending is not necessary 😅

It would have been necessary if it was still in `test_create_assessment.py` file.
```suggestion
def get_hashed_account_id(account_id: str, key: str) -> str:
```